specifications/api/swagger.yaml

```diff
@@ -5733,6 +5733,23 @@ paths:
             schema: *ApiBalanceTransactionPostData
       responses: *responsesBalanceTransaction
 
+  /wallets/{walletId}/transactions-submit:
+    post:
+      operationId: submitTransaction
+      tags: ["Transactions New"]
+      summary: Submit
+      description: |
+        <p align="right">status: <strong>under development</strong></p>
+        Submit a transaction that was already created and signed.
+      parameters:
+        - *parametersWalletId
+      requestBody:
+        required: true
+        content:
+          application/octet-stream:
```

Comment:
If we're going for new endpoint I'd be for considering using `application/json` and accepting CBOR in base64 encoding, which would be consistent with the workflow, i.e.:
 - balance/construct -> returns `transaction` in base64
 - sign (takes `transaction` in base64) -> returns `transaction` in base64
 - submitTransaction (takes `transaction` base64 instead of binary)